.github/workflows/test.yml

```diff
@@ -12,8 +12,10 @@ on:
       - 'README.md'
 jobs:
   unit:
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
     strategy:
+      matrix:
+        os: [macos-latest, windows-latest]
```

Comment:
For feature parity, I've added the matrix build here, but imo it's an overkill. We can reliably only use `unit` on `ubuntu-20.04` (or `ubuntu-latest` for that matter) which does also upload code coverage.

Reply:
This is really cool that this is supported so easily. It does feel like it is overkill at this point, but it is a "nice to have". I am open to either option really.